src/components/HelloWorld/HelloWorld.vue

```diff
@@ -10,29 +12,46 @@ const increment = () => {
 const incrementPinia = () => {
   main.counter += 1
 }
+
+const { locale, t } = useI18n({ messages })
+
+function toggleLocale() {
+  if (locale.value === 'en')
+    locale.value = 'fa'
+  else
+    locale.value = 'en'
+}
 </script>
 
 <template>
+  <div>
+    <button data-testid="toggle-button" @click="toggleLocale">
+      {{ t('button') }} {{ t('changeLocale') }}
+    </button>
+    <span data-testid="local">
+      {{ t('local') }}
+    </span>
+  </div>
   <div class="wrapper">
     <h1 data-testid="counter-pinia">
-      Counter Pinia: {{ main.counter }}
+      {{ t('couterPinia') }} {{ main.counter }}
```

Comment:
how about writing this line like the following:
`{{ t('couterPinia')+ main.counter }}`
I want to know which one looks better

Reply:
They are not the same.
We have extra space on my code. But I prefer the first one syntax.